src/Humanizer/StringValue.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Humanizer
+{
+    public class StringValue : System.Attribute
```

Comment:
I'm not sold on this class! .Net has a variety of this attribute built into it and Humanizer supports them all. Why add a new one?!